from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_downloader(object):
    def setupUi(self, downloader):
        downloader.setObjectName("downloader")
        downloader.resize(1123, 861)
        self.centralwidget = QtWidgets.QWidget(downloader)
        self.centralwidget.setObjectName("centralwidget")
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setGeometry(QtCore.QRect(-1, -1, 1121, 831))
        self.frame.setStyleSheet("background-color: rgb(255, 255, 255);\n"
"border-radius:15px;")
        self.frame.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame.setObjectName("frame")
        self.frame_2 = QtWidgets.QFrame(self.frame)
        self.frame_2.setGeometry(QtCore.QRect(-1, 0, 1121, 101))
        self.frame_2.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.frame_2.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_2.setObjectName("frame_2")
        self.close = QtWidgets.QPushButton(self.frame_2)
        self.close.setGeometry(QtCore.QRect(1080, 10, 31, 28))
        self.close.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.close.setStyleSheet("background-color: rgb(193, 0, 0);\n"
"border-radius:8px;\n"
"")
        self.close.setText("")
        self.close.setObjectName("close")
        self.label = QtWidgets.QLabel(self.frame_2)
        self.label.setGeometry(QtCore.QRect(10, 10, 61, 61))
        self.label.setText("")
        self.label.setScaledContents(True)
        self.label.setObjectName("label")
        self.label_10 = QtWidgets.QLabel(self.frame_2)
        self.label_10.setGeometry(QtCore.QRect(70, 20, 151, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway")
        font.setPointSize(15)
        self.label_10.setFont(font)
        self.label_10.setScaledContents(True)
        self.label_10.setObjectName("label_10")
        self.label_9 = QtWidgets.QLabel(self.frame_2)
        self.label_9.setGeometry(QtCore.QRect(10, 20, 55, 61))
        self.label_9.setText("")
        self.label_9.setPixmap(QtGui.QPixmap("../Pictures/icons8-download-96.png"))
        self.label_9.setScaledContents(True)
        self.label_9.setObjectName("label_9")
        self.label_15 = QtWidgets.QLabel(self.frame_2)
        self.label_15.setGeometry(QtCore.QRect(540, 0, 111, 101))
        self.label_15.setText("")
        self.label_15.setPixmap(QtGui.QPixmap("../Pictures/icons8-youtube-512.png"))
        self.label_15.setScaledContents(True)
        self.label_15.setObjectName("label_15")
        self.min = QtWidgets.QPushButton(self.frame_2)
        self.min.setGeometry(QtCore.QRect(1000, 10, 31, 28))
        self.min.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.min.setStyleSheet("background-color: rgb(0, 229, 0);\n"
"border-radius:8px;")
        self.min.setText("")
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap("../YOUTUBE_DOWNLOADER/Pictures/icons8-subtract-90.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
        self.min.setIcon(icon)
        self.min.setObjectName("min")
        self.max = QtWidgets.QPushButton(self.frame_2)
        self.max.setGeometry(QtCore.QRect(1040, 10, 31, 28))
        self.max.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.max.setStyleSheet("background-color: rgb(255, 255, 0);\n"
"border-radius:8px;")
        self.max.setText("")
        self.max.setIcon(icon)
        self.max.setObjectName("max")
        self.frame_3 = QtWidgets.QFrame(self.frame)
        self.frame_3.setGeometry(QtCore.QRect(0, 90, 191, 741))
        self.frame_3.setStyleSheet("background-color: rgb(255, 0, 0);")
        self.frame_3.setFrameShape(QtWidgets.QFrame.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Raised)
        self.frame_3.setObjectName("frame_3")
        self.pushButton = QtWidgets.QPushButton(self.frame_3)
        self.pushButton.setGeometry(QtCore.QRect(0, 10, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton.setFont(font)
        self.pushButton.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton.setToolTip("")
        self.pushButton.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton.setIconSize(QtCore.QSize(60, 50))
        self.pushButton.setObjectName("pushButton")
        self.pushButton_2 = QtWidgets.QPushButton(self.frame_3)
        self.pushButton_2.setGeometry(QtCore.QRect(0, 90, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_2.setToolTip("")
        self.pushButton_2.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton_2.setIconSize(QtCore.QSize(60, 50))
        self.pushButton_2.setObjectName("pushButton_2")
        self.pushButton_3 = QtWidgets.QPushButton(self.frame_3)
        self.pushButton_3.setGeometry(QtCore.QRect(0, 170, 191, 61))
        font = QtGui.QFont()
        font.setFamily("Raleway SemiBold")
        font.setPointSize(14)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_3.setToolTip("")
        self.pushButton_3.setStyleSheet("background-color: rgb(0, 0, 0);\n"
"color: rgb(255, 0, 0);\n"
"border-radius:0px;")
        self.pushButton_3.setIconSize(QtCore.QSize(60, 60))
        self.pushButton_3.setObjectName("pushButton_3")
        self.label_8 = QtWidgets.QLabel(self.frame_3)
        self.label_8.setGeometry(QtCore.QRect(0, 710, 191, 21))
        self.label_8.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_8.setText("")
        self.label_8.setScaledContents(True)
        self.label_8.setObjectName("label_8")
        self.label_11 = QtWidgets.QLabel(self.frame_3)
        self.label_11.setGeometry(QtCore.QRect(0, 680, 191, 21))
        self.label_11.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_11.setText("")
        self.label_11.setScaledContents(True)
        self.label_11.setObjectName("label_11")
        self.label_12 = QtWidgets.QLabel(self.frame_3)
        self.label_12.setGeometry(QtCore.QRect(0, 650, 191, 21))
        self.label_12.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_12.setText("")
        self.label_12.setScaledContents(True)
        self.label_12.setObjectName("label_12")
        self.label_13 = QtWidgets.QLabel(self.frame_3)
        self.label_13.setGeometry(QtCore.QRect(0, 620, 191, 21))
        self.label_13.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_13.setText("")
        self.label_13.setScaledContents(True)
        self.label_13.setObjectName("label_13")
        self.label_14 = QtWidgets.QLabel(self.frame_3)
        self.label_14.setGeometry(QtCore.QRect(0, 590, 191, 21))
        self.label_14.setStyleSheet("background-color: rgb(0, 0, 0);")
        self.label_14.setText("")
        self.label_14.setScaledContents(True)
        self.label_14.setObjectName("label_14")
        self.lineEdit = QtWidgets.QLineEdit(self.frame)
        self.lineEdit.setGeometry(QtCore.QRect(210, 180, 761, 71))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(15)
        self.lineEdit.setFont(font)
        self.lineEdit.setStyleSheet("background-color: rgb(232, 232, 232);\n"
"border-radius:15px;")
        self.lineEdit.setObjectName("lineEdit")
        self.pushButton_4 = QtWidgets.QPushButton(self.frame)
        self.pushButton_4.setGeometry(QtCore.QRect(210, 390, 211, 61))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semibold")
        font.setPointSize(18)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setCursor(QtGui.QCursor(QtCore.Qt.PointingHandCursor))
        self.pushButton_4.setStyleSheet("background-color: rgb(255, 0, 0);\n"
"border-radius:15px;")
        self.pushButton_4.setObjectName("pushButton_4")
        self.toolButton = QtWidgets.QToolButton(self.frame)
        self.toolButton.setGeometry(QtCore.QRect(980, 290, 71, 51))
        self.toolButton.setStyleSheet("background-color: rgb(232, 232, 232);\n"
"border-radius:15px;")
        self.toolButton.setObjectName("toolButton")
        self.lineEdit_2 = QtWidgets.QLineEdit(self.frame)
        self.lineEdit_2.setGeometry(QtCore.QRect(210, 270, 761, 71))
        font = QtGui.QFont()
        font.setFamily("Segoe UI Semilight")
        font.setPointSize(15)
        self.lineEdit_2.setFont(font)
        self.lineEdit_2.setStyleSheet("background-color: rgb(232, 232, 232);\n"
"border-radius:15px;")
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.label_2 = QtWidgets.QLabel(self.frame)
        self.label_2.setGeometry(QtCore.QRect(200, 520, 321, 241))
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap("../Pictures/Fast loading-amico.svg"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        downloader.setCentralWidget(self.centralwidget)
        self.statusbar = QtWidgets.QStatusBar(downloader)
        self.statusbar.setObjectName("statusbar")
        downloader.setStatusBar(self.statusbar)

        self.retranslateUi(downloader)
        self.close.clicked.connect(downloader.close) # type: ignore
        self.min.clicked.connect(downloader.showMinimized) # type: ignore
        self.max.clicked.connect(downloader.showNormal) # type: ignore
        QtCore.QMetaObject.connectSlotsByName(downloader)

    def retranslateUi(self, downloader):
        _translate = QtCore.QCoreApplication.translate
        downloader.setWindowTitle(_translate("downloader", "MainWindow"))
        self.close.setToolTip(_translate("downloader", "close"))
        self.label_10.setText(_translate("downloader", "Downloader"))
        self.min.setToolTip(_translate("downloader", "minimize"))
        self.max.setToolTip(_translate("downloader", "normal"))
        self.pushButton.setText(_translate("downloader", "Home"))
        self.pushButton.setShortcut(_translate("downloader", "Home"))
        self.pushButton_2.setText(_translate("downloader", "Profile"))
        self.pushButton_3.setText(_translate("downloader", "About Me"))
        self.lineEdit.setPlaceholderText(_translate("downloader", "type the Link"))
        self.pushButton_4.setText(_translate("downloader", "Download"))
        self.toolButton.setText(_translate("downloader", "..."))
        self.lineEdit_2.setPlaceholderText(_translate("downloader", "save to..."))
